tests/integration/conftest.py

```diff
@@ -0,0 +1,124 @@
+import pytest
+import os
+
+from subprocess import call
+from twindb_backup.destination.s3 import S3
+
+BUCKET = 'twindb-backup-test-travis-%s' % os.environ['TRAVIS_JOB_NUMBER']
```

Comment:
if we run the integration test locally `TRAVIS_JOB_NUMBER` won't exist. Should we handle it and replace with a random number or something?